uqcsbot/scripts/news.py

```diff
@@ -0,0 +1,39 @@
+from uqcsbot import bot, Command
+from uqcsbot.utils.command_utils import loading_status
+from typing import Tuple, Dict, List
+
+import feedparser
+
+ARTICLES_TO_POST = 5
+URL = "http://feeds.feedburner.com/TechCrunch/"
+
+def get_tech_crunch_data() -> List[Dict[str, str]]:
+    """
+    Returns data from TechCrunch RSS feed
+    """
+    return feedparser.parse(URL).entries
+
+def get_data_from_article(news: List[Dict[str, str]], index: int) -> Tuple[str, str]:
+    """
+    Returns the title of the article and the link
+
+    Tuple returned: (title, url)
+    """
+    return (news[index]['title'], news[index]['link'])
+
+@bot.on_command("news")
```

Comment:
This feels a bit too specific for a `!new` command. Maybe in future if it supported multiple news sources, but I think `!techcrunch` would be more suitable at the moment.

Reply:
I also think this is a good idea. I'll link it to the `!techcrunch` command for now. But in the future when I have more time I would like to accept arguments such as `!news techcrunch` and `!news 9news' etc and post headlines for chosen news sites